### PR TITLE
Code 222 set up testing and optional

### PR DESCRIPTION
**Pull request by ArBridgeman**

### CODE REVIEW

File: jellyfin-helpers/config/plan_workouts.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ plan_workouts:
     project: Active tasks
     section: ariel_workouts
     task_priority: 2
+    task_priority_optional: 1
```
Comment (ArBridgeman) [inline jellyfin-helpers/config/plan_workouts.yaml:15]: @AkagiShigeru can you verify that this is correct? think the api doesn't match the UI values.
For non-optional ones (default), I want a "blue" coloring &, for optional ones, a "grey" coloring.

**Comment by ArBridgeman** [comment]
@AkagiShigeru seems like the todoist api might be having issues (see utilities). Can you double-check?

File: jellyfin-helpers/config/plan_workouts.yaml (hunk 1 above)
Comment (AlexSPhoenixGames) [reply jellyfin-helpers/config/plan_workouts.yaml:15]: Yes, that should be correct as specified.

Comment (AkagiShigeru) [reply jellyfin-helpers/config/plan_workouts.yaml:15]: Yes, that should be correct as is.

**Review by AkagiShigeru — APPROVED** [review]
looks fine to me, integration test fixed in next PR as discussed.

**Pull request merged**